src/mgmt/cli_mgmt.c

```diff
@@ -534,25 +519,19 @@ attach(const char *name, int npsrbs, struct psr_buf *psrbs,
 		goto out;
 	}
 
-	d_getenv_bool("DAOS_PMIXLESS", &pmixless);
-	if (pmixless) {
-		if (psrbs == NULL)
-			rc = get_attach_info(name, &sys->sy_npsrs,
-					     &sys->sy_psrs);
-		else
-			rc = get_attach_info_from_buf(npsrbs, psrbs,
-						      &sys->sy_npsrs,
-						      &sys->sy_psrs);
-		if (rc != 0)
-			goto err_sys;
-		if (sys->sy_npsrs < 1) {
-			D_ERROR(">= 1 PSRs required: %d\n", sys->sy_npsrs);
-			goto err_psrs;
-		}
+	if (psrbs == NULL)
+		rc = get_attach_info(name, &sys->sy_npsrs, &sys->sy_psrs);
+	else
+		rc = get_attach_info_from_buf(npsrbs, psrbs, &sys->sy_npsrs,
+					      &sys->sy_psrs);
```

Comment:
OK, so by this we add the psrs to group (by attach_group) then need not previous attach info file right? so client can know the psr uris, then cart can do the addr resolution...